### DISCUSSION TITLE
Duino coins value

### DISCUSSION THREAD

**Discussion by rudraksha007** [revoxhere/duino-coin#1020] [General]
Hello, I am a new user here but the from I joined this, the value of the coins is getting lower and lower by the time. I know, it is written that it's a fun project and not give much profits to user but I wanna that, will it ever gain value?

**Comment by gsrbr** [comment]
Hello @rudraksha007  , like any other currency, the duco has its value defined by the Law of Supply and Demand, that is, when demand is high and supply is low, the currency appreciates and the same goes for the opposite, when the supply is very large and demand is low the currency devalues... some factors influence supply and demand, such as usability, security, stability and predictability
